news/service/db.py

```diff
@@ -0,0 +1,18 @@
+from pymongo import MongoClient
+
+_DB_CLIENT = MongoClient("mongodb://news-db:27017/news-db")
+_DB_CLUSTERED_ARTICLES = _DB_CLIENT['news-db']['clusteredarticles']
+
+
+def fetch_n_clusters_before_time(timestamp: int, n_clusters: int):
```

Comment:
i feel like both methods can be combined into one

Reply:
sure, i just wanted to make it more verbose